scripts/new_bare_repo

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
+
+EOF
+}
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+
+elif [ $# -lt 2 ]; then
+   echo 'Need to specify the hutch and name of the proposed bare repo' >&2
+   usage
+   exit 1
+
+fi
+
+HUTCH=$1
+REPO=$2
+CWD=$(PWD)
+
+
+if [ -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/" ] && [ ! -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git" ] && [${CWD} == *${REPO}* ]
+then
+    git init
+    cp /cds/sw/tools/bin/eco_tools/gitignore.template .gitignore
+    git add *.cfg Makefile .gitignore
+    git commit -m "Initial commit"
+    git-bare-repo /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git
+    git remote add origin /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git
+    git push -u origin master
+elif [ ! -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/" ]
+    echo "The directory /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/ does not exist!"
```

Comment:
A general principle I like to follow is that of "exiting early" (i.e., when conditions aren't right).

That is to say, I think it would make more sense to check all directories at the start of the script before doing the bare repo creation.

```
# check if hutch directory doesn't exist (and exit if so)
# check if repo directory already exists (and exit if so)
# check if "REPO" not in $PWD (and exit if so)
# otherwise, do the thing
```